src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -779,14 +779,24 @@ private static void ThrowOnErrorSymbol(ISymbol symbol)
 				}
 
 				var isKnownNamespace = ns?.Prefix?.HasValue() ?? false;
-				var fullName = isKnownNamespace && ns != null ? ns.Prefix + ":" + type.Name : type.Name;
 
-				return _findType!(fullName);
-			}
-			else
-			{
-				return null;
+				if (strictSearch)
+				{
+					if(isKnownNamespace && ns != null)
```

Comment:
when `strictSearch == true` if this condition (`isKnownNamespace && ns != null`) is not meet it will return `null` while on  `strictSearch == false` it returns `_findType!(type.Name);` is this expected?